packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -199,6 +199,120 @@ describe('Account', () => {
       expect(accountHistories.length).toStrictEqual(1)
     })
   })
+
+  it('should listAccountHistory with options owner mine block height and txn', async () => {
+    const options = {
+      maxBlockHeight: 102,
+      txn: 2
+    }
+    const accountHistories = await client.account.listAccountHistory('mine', options)
+    expect(accountHistories.length).toBeGreaterThan(0)
+    for (let i = 0; i < accountHistories.length; i += 1) {
+      const accountHistory = accountHistories[i]
+      expect(accountHistory.blockHeight).toBeLessThanOrEqual(options.maxBlockHeight)
+      if (accountHistory.blockHeight === options.maxBlockHeight) {
+        expect(accountHistory.txn).toBeLessThanOrEqual(options.txn)
+      }
+    }
+  })
+
+  // If you don't set maxBlockHeight, it will consider the last block height
+  it('should listAccountHistory with owner mine options txn', async () => {
+    const options = {
+      txn: 2
+    }
+    const accountHistories = await client.account.listAccountHistory('mine', options)
+    expect(accountHistories.length).toBeGreaterThan(0)
+    for (let i = 0; i < accountHistories.length; i += 1) {
+      const accountHistory = accountHistories[i]
+      expect(accountHistory.blockHeight).toBeLessThanOrEqual(103)
+      if (accountHistory.blockHeight === 103) {
+        expect(accountHistory.txn).toBeLessThanOrEqual(options.txn)
+      }
+    }
+  })
+
+  it('should listAccountHistory with owner all options block height and txn', async () => {
+    const options = {
+      maxBlockHeight: 102,
+      txn: 1
+    }
+    const accountHistories = await client.account.listAccountHistory('all', options)
+    expect(accountHistories.length).toBeGreaterThan(0)
+    for (let i = 0; i < accountHistories.length; i += 1) {
+      const accountHistory = accountHistories[i]
+      expect(accountHistory.blockHeight).toBeLessThanOrEqual(options.maxBlockHeight)
+      if (accountHistory.blockHeight === options.maxBlockHeight) {
+        expect(accountHistory.txn).toBeLessThanOrEqual(options.txn)
+      }
+    }
+  })
+
+  it('should listAccountHistory with owner all options block height, type and txn', async () => {
+    const options = {
+      maxBlockHeight: 103,
+      txn: 2,
+      txtype: DfTxType.MINT_TOKEN
+    }
+    const accountHistories = await client.account.listAccountHistory('all', options)
+    expect(accountHistories.length).toBeGreaterThan(0)
+    for (let i = 0; i < accountHistories.length; i += 1) {
+      const accountHistory = accountHistories[i]
+      expect(accountHistory.blockHeight).toBeLessThanOrEqual(options.maxBlockHeight)
+      if (accountHistory.blockHeight === options.maxBlockHeight) {
+        expect(accountHistory.txn).toBeLessThanOrEqual(options.txn)
+      } else {
+        expect(accountHistory.txn).toBeGreaterThanOrEqual(options.txn)
```

Comment:
ideally this(`accountHistory.txn`) could be any valid number given that `accountHistory.blockHeight != 103` ?